File: ClalitTest/ClientApp/src/components/Currency.js
import React from 'react';

export function Currency({currency}) {
  return (
    <tr key={currency.currencyCode}>
        <td>{currency.name}</td>
        <td>{currency.unit}</td>
        <td>{currency.currencyCode}</td>
        <td>{currency.country}</td>
        <td>{currency.rate}</td>
        <td>{currency.change}</td>
    </tr>
  );
}

File: ClalitTest/Currency.cs
namespace ClalitTest
{
    public class Currency
    {
        public string Name { get; set; } = null!;
        public int Unit { get; set; }
        public string Country { get; set; } = null!;
        public string CurrencyCode { get; set; } = null!;
        public double Rate { get; set; }
        public double Change { get; set; }
    }
}